src/Func.h

```diff
@@ -921,15 +921,18 @@ class Func {
                               const std::vector<ExternFuncArgument> &params,
                               Type t,
                               int dimensionality,
-                              NameMangling mangling = NameMangling::Default) {
-        define_extern(function_name, params, std::vector<Type>{t}, dimensionality, mangling);
+                              NameMangling mangling = NameMangling::Default,
+                              bool uses_old_buffer_t = false /* currently does nothing. See PR #1899. */) {
```

Comment:
The default will be false in the main PR. In this PR it doesn't really matter what it is.